src/Symfony/Component/Config/Resource/FileResource.php

```diff
@@ -56,11 +62,7 @@ public function getResource()
      */
     public function isFresh($timestamp)
     {
-        if (false === $this->resource || !file_exists($this->resource)) {
-            return false;
-        }
-
-        return filemtime($this->resource) <= $timestamp;
+        return file_exists($this->resource) && filemtime($this->resource) <= $timestamp;
```

Comment:
I think @Tobion meant: `return file_exists($this->resource) && @filemtime($this->resource) <= $timestamp;`.